internal/core/command/device.go

```diff
@@ -64,17 +67,18 @@ func commandByDeviceID(deviceID string, commandID string, body string, queryPara
 
 	if c.String() == (contract.Command{}).String() {
 		errMsg := fmt.Sprintf("Command with id '%v' does not belong to device with id '%v'.", commandID, deviceID)
-		LoggingClient.Error(errMsg)
+		loggingClient.Error(errMsg)
 		return errMsg, http.StatusNotFound
 	}
 
-	return commandByDevice(d, c, body, queryParams, isPutCommand, ctx)
+	return commandByDevice(d, c, body, queryParams, isPutCommand, ctx, loggingClient)
 }
 
-func commandByNames(dn string, cn string, body string, queryParams string, isPutCommand bool, ctx context.Context) (string, int) {
+func commandByNames(dn string, cn string, body string, queryParams string, isPutCommand bool, ctx context.Context,
+	loggingClient logger.LoggingClient) (string, int) {
```

Comment:
Commented here; applies elsewhere.

For readability and consistency (not yet, but I'm pushing towards a readable standard), when folding function/method declarations, place one parameter on each line.  For example:

```
func commandByNames(
    dn string, 
    cn string, 
    body string, 
    queryParams string, 
    isPutCommand bool, 
    ctx context.Context,
    loggingClient logger.LoggingClient) (string, int) {
```